### ISSUE TITLE
the cursor on the monitor for open the start menu

### ISSUE THREAD

**Issue by Garze23** [valinet/ExplorerPatcher#3898]
i use it
![image](https://github.com/user-attachments/assets/d82f0dd8-15da-4e85-b331-5a3cc8891f79)
but it's not work
![image](https://github.com/user-attachments/assets/d947045e-ee4c-4c95-9520-891e46cccb66)
-x86-64
-french

**Comment by milesprower2293** [comment]
Did you restart explorer?

**Comment by Garze23** [comment]
Yes, i had this option since 1 years, so yes i restart the explorer, this problème exist for a long time for me

**Comment by Garze23** [comment]
?

**Comment by Garze23** [comment]
?

**Comment by pyrates999** [comment]
can you post your version of EP?

**Comment by Garze23** [comment]
![image](https://github.com/user-attachments/assets/12045291-02c7-48c2-a080-5fcde2e72106)

**Comment by pyrates999** [comment]
when you press the windows key, which monitor does the start menu open on?

**Comment by Garze23** [comment]
on the main screen configured in Windows settings

**Comment by pyrates999** [comment]
and what screen are you expecting the start menu to open on?  and why that screen?

**Comment by Garze23** [comment]
For example, my cursor is on my right-hand screen, which is not my main screen, so the start menu opens on the main screen when it's supposed to open on the screen where the cursor is.

**Comment by pyrates999** [comment]
please wait for a developer to respond.

**Comment by dissonator** [comment]
same problem
